packages/admin/src/tabs/RouterTabs.tsx

```diff
@@ -63,63 +71,58 @@ function RouterTabsComponent({
         rearrangedChildren.push(defaultPathChild);
     }
 
+    React.useEffect(() => {
+        if (stackApi && stackSwitchApi) {
+            // When inside a Stack show only the last TabBar
+            const ownSwitchIndex = stackSwitchApi.id ? stackApi.switches.findIndex((i) => i.id === stackSwitchApi.id) : -1;
+            const nextSwitchShowsInitialPage = stackApi.switches[ownSwitchIndex + 1] && stackApi.switches[ownSwitchIndex + 1].isInitialPageActive;
+
+            setShouldShowTabBar(ownSwitchIndex === stackApi.switches.length - (nextSwitchShowsInitialPage ? 2 : 1));
+        }
+    }, [stackApi, stackSwitchApi]);
```

Comment:
Previously, this was not a state, but simply a variable. Why did you change that? Doing so causes an additional render when `stackApi` or `stackSwitchApi` changes.